docs/user-guide/mvd-configuration.md

```diff
@@ -175,30 +174,91 @@ For more information, see [Logging Utility](../extend/extend-desktop/mvd-logutil
+Attributes that control ZSS are in the agent object. For example, *agent.https.port* is the TCP port that ZSS will listen on to be contacted by the App Server. Define this in the configuration file as a value between 1024-65535. Similarly, if specified, *agent.https.ipAddresses* will be used to determine which IP addresses the server should bind to. Only the first value of the array is used. It can either be a hostname or an ipv4 address.
 
 Example of the agent body:
 ```
-  "agent": {
-    "host": "localhost",
-    "http": {
-      "ipAddresses": ["127.0.0.1"],
-      "port": 8542
-    }
-  }
+components:
+  zss:
+    enabled: true
+    port: "8542"
+    certificate:
+      keystore:
+        alias: zss
+      pem:
+        key: /my/keystore/localhost/localhost.keystore.app-server.key
+        certificate: /my/keystore/localhost/localhost.keystore.app-server.cer-ebcdic
+      agent:
+        host: "localhost"
+        https:
+          ipAddresses:
+            - "127.0.0.1"
 ```
 
 ### Connecting App Server to ZSS
 
-When running the App Server, simply specify a few flags to declare which ZSS instance the App Server will proxy ZSS requests to:
+When running App Server and ZSS together, parameters are needed to specify the location for App Server to find ZSS at. This is needed regardless of if ZSS and App server are running on the same system or not.
+When using instance.env, the parameters `ZOWE_ZSS_SERVER_PORT` and `ZWED_agent_https_host` specify the port and host to find ZSS at, and are read by both the App server and ZSS.
+When using zowe.yaml, app-server will read the zss component section if it exists, or will look for the agent section within the app-server configuration if not. Below is are two examples of specifying the host and port.
+
+```
+components:
+  app-server:
+    enabled: true
+    port: "8544"
+    certificate:
+      keystore:
+        alias: app-server
+      pem:
+        key: /my/keystore/localhost/localhost.keystore.app-server.key
+        certificate: /my/keystore/localhost/localhost.keystore.app-server.cer-ebcdic
+      agent:
+        host: "localhost"
+        https:
+          port: 8542
+```
+
+Or with both components,
+
+```
+components:
+  app-server:
+    enabled: true
+    port: "8544"
+    certificate:
+      keystore:
+        alias: app-server
+      pem:
+        key: /my/keystore/localhost/localhost.keystore.app-server.key
+        certificate: /my/keystore/localhost/localhost.keystore.app-server.cer-ebcdic
+components:
+  zss:
+    enabled: true
+    port: "8542"
+    certificate:
+      keystore:
+        alias: zss
+      pem:
+        key: /my/keystore/localhost/localhost.keystore.app-server.key
+        certificate: /my/keystore/localhost/localhost.keystore.app-server.cer-ebcdic
+      agent:
+        host: "localhost"
+        https:
```

Comment:
Just double checking that "host" and "https" belong in "components.zss.agent" not in "components.zss"